includes/class-image-optimisation.php

```
@@ -1372,6 +1443,22 @@ public function add_delay_load_img( $buffer ) {
 			$exclude_img_count  = $image_optimisation['excludeFirstImages'] ?? 0;
 			$exclude_imgs       = array();
 
+			$enable_video_placeholder = ! empty( $image_optimisation['enableVideoPlaceholder'] );
```

PHPMD (2.15.0)

[warning] Avoid excessively long variable names like $enable_video_placeholder. Keep variable name length under 20. (undefined)

(LongVariable)